tox.ini

```diff
@@ -47,8 +47,13 @@ commands =
   coverage3 report
 
 [testenv:docs]
+# Editable mode breaks macOS: https://github.com/sphinx-doc/sphinx/issues/10943
+usedevelop = False
 envdir = .tox/docs
 basepython = python3
+deps =
+  -r{toxinidir}/requirements-dev.txt
+  .
```

Comment:
I'm not really sure why the `.` was necessary - I thought Sphinx normally would install that for us. It means to install optimization itself.